lib/plugins/acts_as_journalized/lib/acts_as_journalized.rb

```diff
@@ -172,6 +132,12 @@ def journalized_event_hash(options)
           { description: :notes }.reverse_merge options
         end
       end
+
+      # We can't use a :has_many relation as this would try to delete journals upon deleting the
+      # journalized object (there is nothing like "dependent: ignore").
```

Comment:
Nullify will also change the journals (setting the foreign key reference to `NULL`). I have some problems here:

1. The `has_many :journals`  would have deleted those records anyway (aggregated journals are only a subset of those)
2. The way we are aggregating the journals might even break when trying to delete the journals (it did for me once, but that was before @ulferts and I refactored the SQL, not sure what would happen now)
3. We even blend some information from different journals, so that makes it an even worse idea to delete records based on that

I now feel that `has_many`, might for multiple reasons not be the right choice here...